ibis/backends/duckdb/__init__.py

```diff
@@ -74,6 +104,10 @@ class Backend(AlchemyCrossSchemaBackend, CanCreateSchema):
     compiler = DuckDBSQLCompiler
     supports_create_or_replace = True
 
+    @cached_property
```

Comment:
Perhaps only `@property` is okay? We're only saving on construction, which shouldn't be expensive. If you still want to avoid it, a `_settings` instance attribute would be simpler than a cached property IMO.